### ISSUE TITLE
Not able to add custom refiners

### ISSUE THREAD

**Issue by srinutamada** [pnp/sp-dev-solutions#416]
I am using search box, results and refiners web part on my page. The search is restricted to only 1 list. Below are the issues I am facing
1. Default search results showing properly and data is coming from only 1 list. But if I search any content on the page, I see data from other lists and libraries. When I edited the webpart at that moment, it started showing specific to the list I targeted.
2. Refiners are not working. I am not able to add customer refiners which I have other than date, size file types. Is there any way to add custom refiners?
I add all refiners to select search query also to make sure I am not missing anything.

Please help me in resolving above issues.

**Comment by wobba** [comment]
If you let us know how your query looks and how you have mapped the crawled properties to managed properties and ensure you have values we can assist. Also remember to re-index items after you map a cp to an mp (needed only once if mapping occurs after the data exists).

Basically without repro steps and details on your setup it's very hard to help out.

I recommend you replicate your setup for classic web parts, and if you get the same error it's an error in your setup and not the web parts.

**Comment by srinutamada** [comment]
I am able to resolve both the issues
1. I added path to refiners instead of adding to query string. After making the change I am able to search content with in a list
2. The issue with refiner web part is, edit refiners screen is not coming properly in Chrome. I was not able to see available refiners but on mouse over I was able to see full refiner name. Now I configured them properly

Now my search is working the way I wanted.
